src/app/launcher/import-app/gitprovider-importapp-step/gitprovider-importapp-step.component.html

```diff
@@ -59,30 +61,30 @@ <h2>GitHub Information</h2>
             </div>
             <div class="form-group">
               <label for="ghRepo" class="col-sm-2 control-label">Repository</label>
-              <div class="col-sm-10"
-                   [ngClass]="{'has-error': launcherComponent.summary?.gitHubDetails?.repositoryAvailable === false}">
-                <input id="ghRepo" class="form-control" name="ghRepo" type="text"
-                       placeholder="Enter a unique name for your repository"
-                       [disabled]="launcherComponent.summary?.gitHubDetails?.organization === undefined"
-                       [(ngModel)]="launcherComponent.summary.gitHubDetails.repository"
-                       (ngModelChange)="updateGitHubSelection()"
-                       (keyup)="validateRepo()">
-                <span class="help-block"
-                      *ngIf="launcherComponent.summary?.gitHubDetails?.repositoryAvailable === false">
-                  Duplicate Name: {{duplicateNameMessage}}
-                </span>
+              <div class="col-sm-10">
+                <select id="ghRepo" class="form-control" name="ghRepo" placeholder="Select Repository"
+                        [disabled]="launcherComponent.summary?.gitHubDetails?.organization === undefined || launcherComponent.summary?.gitHubDetails?.repositoryList?.length === 0"
+                        [(ngModel)]="launcherComponent.summary.gitHubDetails.repository"
+                        (ngModelChange)="updateGitHubSelection()">
+                  <option [ngValue]="undefined" selected>Select</option>
+                  <option [ngValue]="repo"
+                          *ngFor="let repo of launcherComponent.summary.gitHubDetails.repositoryList">
+                    {{repo}}
+                  </option>
+                </select>
               </div>
             </div>
           </div>
         </div>
       </div>
     </div>
-    <div class="container-fluid f8launcher-provider-card-toolbar">
+    <!-- TODO : Need to be implemented based on UXD -->
+    <!-- <div class="container-fluid f8launcher-provider-card-toolbar">
       <pfng-toolbar [config]="toolbarConfig"
                     (onFilterChange)="filterChanged($event)"
                     (onSortChange)="sortChanged($event)">
       </pfng-toolbar>
-    </div>
+    </div> -->
```

Comment:
You can remove the toolbar. We won't be using filter and sort with the latest UX design.

Please remove the filter, sort, and toolbar imports from patternfly-ng found in gitprovider-importapp-step.component.ts and launcher.module.ts.

Reply:
Sure Dan